testdata/webhooks/note_merge_request.json

```diff
@@ -110,11 +136,10 @@
     },
     "work_in_progress": false,
     "assignee": {
-      "id": 16,
       "name": "User1",
       "username": "user1",
-      "email": "user1@example.com",
       "avatar_url": "http://www.gravatar.com/avatar/e64c7d89f26bd1972efa854d13d7dd61?s=40\u0026d=identicon"
-    }
+    },
+    "detailed_merge_status": "checking"
   }
-}
+}
```

Comment:
Please make sure to end the last line with a newline char...

Reply:
Fixed